src/panels/lovelace/editor/view-editor/hui-view-visibility-editor.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+import "@polymer/paper-input/paper-input";
+
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { configElementStyle } from "../config-elements/config-elements-style";
+import { ShowViewConfig } from "../../../../data/lovelace";
+
+import "../../components/hui-theme-select-editor";
+
+import { User } from "../../../../data/user";
+import memoizeOne from "memoize-one";
+import { compare } from "../../../../common/string/compare";
+import { HaSwitch } from "../../../../components/ha-switch";
+
+declare global {
+  interface HASSDomEvents {
+    "view-visibility-changed": {
+      visible: ShowViewConfig[];
+    };
+  }
+}
+
+@customElement("hui-view-visibility-editor")
+export class HuiViewVisibilityEditor extends LitElement {
+  static get styles(): CSSResult {
+    return css`
+      .panel {
+        color: var(--secondary-text-color);
```

Comment:
This should be `primary-color` (so just default)

Reply:
And move `styles` to the bottom of the class.

Reply:
@bramkragten I tried to create a view that will look as mockup created in https://github.com/home-assistant/home-assistant-polymer/issues/4139
So I can safely remove that class (if `primary-color` is the default)?

Reply:
yes

Reply:
ok removed